packages/BetterPhpDocParser/ValueObject/Parser/BetterTokenIterator.php

```diff
@@ -103,7 +103,7 @@ public function currentPosition(): int
     }
 
     /**
-     * @return array<array{0: string, 1: int}>
+     * @return array<int, array{string, int, int}>
      */
     public function getTokens(): array
```

Comment:
why does this class need `privatesAccessor` to access its own properties?

Reply:
I have limited knowledge about this, probably better to verify on separate PR for improvement when possible :+1:

Reply:
👍

Reply:
I see now why its done. the class is setting the private properties of its base-class, which would otherwise not be accessible